lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -111,27 +145,45 @@
 
         AccountSettingsFieldViews.AuthFieldView = FieldViews.LinkFieldView.extend({
 
+            fieldTemplate: field_social_link_template,
+
+            className: function () {
+                return 'u-field u-field-social u-field-' + this.options.valueAttribute;
+            },
+
             initialize: function (options) {
                 this.options = _.extend({}, options);
                 this._super(options);
                 _.bindAll(this, 'redirect_to', 'disconnect', 'successMessage', 'inProgressMessage');
             },
 
             render: function () {
-                var linkTitle;
+                var linkTitle, linkClass, subTitle;
                 if (this.options.connected) {
-                    linkTitle = gettext('Unlink');
+                    linkTitle = gettext('Unlink this account');
+                    linkClass = 'social-field-linked';
+                    subTitle = gettext(
+                        'Your ' + this.options.title + ' account can be used to login into your edX account.'
+                    );
                 } else if (this.options.acceptsLogins) {
-                    linkTitle = gettext('Link')
+                    linkTitle = gettext('Link your Account');
+                    linkClass = 'social-field-unlinked';
+                    subTitle = gettext(
+                        'Link your ' + this.options.title + ' account so you can easily log into edX ' +
+                        'using your ' + this.options.title + ' account details.'
+                    );
                 } else {
-                    linkTitle = ''
+                    linkTitle = '';
+                    linkClass = '';
```

Comment:
You could set these values when you declare the variables at the top of the function, then you wouldn't need this else block.